src/odb/src/codeGenerator/schema.json

```diff
@@ -143,6 +143,181 @@
     },
     {
       "src":"dbAccessPoint.json"
+    },
+    {
+      "name":"dbPowerDomain",
+      "type":"dbObject",
+      "fields":[
+        {
+          "name":"_name",
+          "type":"char *",
```

Comment:
We should use std::string rather than char*

Reply:
_name needs to be a char* due to the dbHashTable but the others were converted to std::string